deploy/helm/charts/Chart.yaml

```diff
@@ -16,11 +16,5 @@ sources:
 - https://github.com/openebs/lvm-localpv
 dependencies:
   - name: crds
-    version: "1.5.0"
-maintainers:
-  - name: prateekpandey14
-    email: prateek.pandey@mayadata.io
-  - name: pawanpraka1
-    email: pawan@mayadata.io
-  - name: iyashu
-    email: yashpal.c1995@gmail.com
+    version: "1.5.1"
```

Comment:
nit: this shouldn't have the double quotes because we're typically not using quotes in chart versions.